f_x: 1140
f_y: 1140
u_0: 0
v_0: 0
w: 2048
h: 1536

roll: 0
pitch: 0
yaw: 0
translation: [0.0, 0.0, 0.0]

frequency: 10
MPC_horizon: 2.0
sampling_interval: 0.01

a_max: 2.0
